src/main/factories/validations/change-password-validation-factory.ts

```diff
@@ -0,0 +1,26 @@
+import { type Validation } from '@/application/protocols'
+import {
+  ValidationComposite,
+  RequiredFieldValidation,
+  PasswordValidation,
+  CompareFieldsValidation
+} from '@/application/validation'
+import { PasswordValidatorAdapter } from '@/infra/validators'
+import { PasswordMismatchError, PasswordRequirementsError } from '@/application/errors'
+
+export const makeChangePasswordValidation = (): ValidationComposite => {
```

Comment:
Excelente implementation. All validators in one place out of controller is good to separation of concerns.

Reply:
yeah i thought it was pretty cool too;